src/com/walmartlabs/lacinia/pedestal2.clj

```diff
@@ -312,21 +313,25 @@
 
   It may also contain keys :app-context and :port (which defaults to 8888).
 
+  You can also define an explicit :host address to your application. Useful when running inside Docker.
+
   This is useful for initial development and exploration, but applications with any more needs should construct
   their service map directly."
   [compiled-schema options]
-  (let [{:keys [api-path ide-path asset-path app-context port]
+  (let [{:keys [api-path ide-path asset-path app-context port host]
```

Comment:
I'm less keen on this part of the change, as the goal with `default-service` is to be a minimal bit of placeholder scaffolding that should always be replaced in a real application. Anything more complex than this would probably not get approved.

Reply:
Makes sense! Thanks for the clarification.